frame/staking/src/migrations.rs

```diff
@@ -129,6 +134,11 @@ pub mod v11 {
 
 		#[cfg(feature = "try-runtime")]
 		fn post_upgrade(_state: Vec<u8>) -> Result<(), &'static str> {
+			frame_support::ensure!(
+				StorageVersion::<T>::get() == crate::Releases::V11_0_0,
```

Comment:
I would compare that directly to V12

Reply:
Why so? This is the v11 migration. Your change should upgrade v11, execute `post_upgrade` and only then move on to the next migration right?

Reply:
I got confused as well; this diff changes two different migrations. It is correct.

Reply:
@ggwpez Ah I know what it is, this was disabled because try-runtime was  not handling things sequentially.